visvis2/cameras/_base.py

```diff
@@ -3,12 +3,55 @@
 
 
 class Camera(WorldObject):
+    """ Abstract base camera.
+    """
+
     def __init__(self):
         super().__init__()
 
         self.matrix_world_inverse = Matrix4()
         self.projection_matrix = Matrix4()
         self.projection_matrix_inverse = Matrix4()
 
+    def update_viewport_size(self, width, height):
```

Comment:
this is just a setter I think, so instead of `update` it should say `set`

Reply:
I agree on using `set`. I figured that there may be camera's that need both width and height. E.g. a camera that keeps its aspect ratio can confine itself to whichever dimension is (relatively) the smallest.